src/CodeGen.elm

```diff
@@ -585,45 +585,64 @@ emitPadding value attrs =
             :: attrs
 
 
-emitBorder : Color -> BorderStyle -> BorderWidth -> List Expression -> List Expression
-emitBorder borderColor borderStyle borderWidth attrs =
-    if borderWidth.top == 0 && borderWidth.right == 0 && borderWidth.bottom == 0 && borderWidth.left == 0 then
-        attrs
+emitBorder : Color -> BorderStyle -> BorderWidth -> BorderCorner -> List Expression -> List Expression
```

Comment:
Since I guess we are both using `elm-format` it must be a Github thing, but I'm wondering while I'm seeing the function completely changed, why you just added a block towards the end. 🧐

Reply:
I'ts because I added `CodeGen.apply` at line 590